### PR TITLE
feat/our-results

### PR DESCRIPTION
**Pull request by henriquecalenzoo**
## Description

Provide a description of what was changed and why it was implemented this way.

## Dev Notes

- If any developer notes are required, list them here. If not, remove the 'Notes' section.

### Steps

1. List all steps in a numbered list

## Screenshots / Videos

Add any relevant screenshots or videos. Any video should be added to the Jira ticket and linked here.

## Developer Checks

- [ ] PR title & commits adhere to [Conventional Commit Specification](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] PR is targeting correct branch, is up-to-date, & no merge conflicts
- [ ] Tested on browser device
- [ ] Tested on Mobile device

### CODE REVIEW

**Review by ramonxm — CHANGES_REQUESTED** [review]

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.74 | ✅ |
| First Contentful Paint | 2.8 s |  |
| Time to Interactive | 4.2 s |  |
| Speed Index | 3.4 s |  |
| Total Blocking Time | 310 ms |  |
| Largest Contentful Paint | 4.6 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.94 | ✅ |
| First Contentful Paint | 1.6 s |  |
| Time to Interactive | 5.3 s |  |
| Speed Index | 2.7 s |  |
| Total Blocking Time | 180 ms |  |
| Largest Contentful Paint | 1.9 s |  |
| Cumulative Layout Shift | 0.044 |  |

    
</details>

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

File: src/styles/theme.tsx [hunk 1]
```diff
@@ -2,6 +2,17 @@ import { DefaultTheme } from 'styled-components';
...
+    mobile_portrait: '320px',
+    mobile_default: '360px',
+    mobile_landscape: '480px',
+    tablet_portrait: '600px',
+    tablet: '768px',
+    tablet_landscape: '801px',
+    laptops_landscape: '1024px',
+    desktop_hd: '1280px',
+    desktop_fullhd: '1600px',
+  },
```
Comment (joaoprferreira) [inline src/styles/theme.tsx:15]: Boaaaa

**Pull request merged**